Relative imports are broken

Relative imports are broken due to unspecified glslify `basedir` option:

`src/shaders/pbr/fragment.glsl`

```glsl
// ...
#pragma glslify: SRGBToLinear = require('../math/srgb-to-linear.glsl')
// ...
```

`src/shaders/math/srgb-to-linear.glsl`

```glsl
// ...
#pragma glslify: export(SRGBtoLinear)
```

Output of `vite build` command:

```
[vite-plugin-glslify:files] Cannot find module '../math/srgb-to-linear.glsl' from './node_modules/vite-plugin-glslify/dist'
file: ./src/shaders/pbr/fragment.glsl
error during build:
Error: Cannot find module '../math/srgb-to-linear.glsl' from './node_modules/vite-plugin-glslify/dist'
    at Function.module.exports [as sync] (./node_modules/glsl-resolve/node_modules/resolve/lib/sync.js:32:11)
    at Depper.resolveSync [as resolve] (./node_modules/glsl-resolve/index.js:33:19)
    at ./node_modules/glslify-deps/sync.js:156:27
    at Array.forEach (<anonymous>)
    at resolveImports (./node_modules/glslify-deps/sync.js:149:13)
    at Depper.add (./node_modules/glslify-deps/sync.js:128:3)
    at Depper.inline (./node_modules/glslify-deps/sync.js:55:15)
    at Object.compile (./node_modules/glslify/index.js:47:23)
    at module.exports.compile (./node_modules/glslify/index.js:22:18)
    at Object.transform (./node_modules/vite-plugin-glslify/dist/index.js:41:64)
```

--- a/src/files_compiler.ts
+++ b/src/files_compiler.ts
@@ -1,3 +1,4 @@
+import path from 'path'
 import { compile } from 'glslify'
 import { Plugin } from 'vite'
 
@@ -9,7 +10,9 @@ export function filesCompiler(extFilter: Filter): Plugin {
     transform(code, id) {
       if (extFilter(id)) {
         return {
-          code: `export default \`${compile(code)}\``
+          code: `export default \`${compile(code, {
+            basedir: path.dirname(id)
+          })}\``
         }
       }
       return null
@@ -18,7 +21,9 @@ export function filesCompiler(extFilter: Filter): Plugin {
       if (!extFilter(ctx.file)) return
       const defaultRead = ctx.read
       ctx.read = async () => {
-        return compile(await defaultRead())
+        return compile(await defaultRead(), {
+          basedir: path.dirname(ctx.file)
+        })
       }
     }
   }

--- a/src/literals_compiler.ts
+++ b/src/literals_compiler.ts
@@ -1,3 +1,4 @@
+import path from 'path'
 import { generate } from 'astring'
 import { walk } from 'estree-walker'
 import { compile } from 'glslify'
@@ -18,7 +19,9 @@ export function literalsCompiler(idFilter: Filter, funcFilter: Filter): Plugin {
         const compileAndOverwrite = (node: any, start: number, end: number) => {
           const target = generate(node)
           try {
-            const compiled = compile(target.replace(/`/g, ''))
+            const compiled = compile(target.replace(/`/g, ''), {
+              basedir: path.dirname(id)
+            })
             s.overwrite(start, end, `\`${compiled}\``)
           } catch (e) {
             this.error(e.message)

